packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -25,6 +25,39 @@ export class FauxClassGenerator {
   private STANDARDOBJECTS_DIR = 'standardObjects';
   private CUSTOMOBJECTS_DIR = 'customObjects';
 
+  // the empty string is used to represent the need for a special case
+  // usually multiple fields with specialized names
+  private static typeMapping: Map<string, string> = new Map([
+    ['string', 'String'],
+    ['double', 'Decimal'],
+    ['reference', ''],
+    ['boolean', 'Boolean'],
+    ['currency', 'Currency'],
+    ['date', 'Date'],
+    ['datetime', 'Datetime'],
+    ['email', 'Email'],
+    ['location', 'Location'],
+    ['percent', 'Decimal'],
+    ['phone', 'String'],
+    ['picklist', 'String'],
+    ['multipicklist', 'String'],
+    ['textarea', 'String'],
+    ['encryptedstring', 'String'],
+    ['url', 'String'],
+    ['id', 'Id'],
+    // note that the mappings below "id" only occur in standard SObjects
+    ['base64', 'Blob'],
+    ['address', 'Address'],
+    ['int', 'Integer'],
+    ['anyType', 'Object'],
+    ['combobox', 'String'],
+    ['time', 'Time'],
+    // TBD what are these mapped to and how to create them
```

Comment:
What is the severity/frequency of these two types occurring?

Reply:
1. I have run all the standard SObjects that I have been able to (I need to try some other perms, etc. to see if I can get some others.   I don't think we still have access to the org that had everything that Josh knew about)....     I can't find these in any of those describes.
2. IF they show up, they will create a "type" for the field that is the name with a capital first.  It "probably" isn't correct, but at least it won't be "undefined" or null...      
3. Since I haven't been able to create these in the standard SObjects, I don't think that the risk is very big - although I still plan to circle back at some point and see if there is a way to create them.